Corona-Warn-App/src/main/java/de/rki/coronawarnapp/release/NewReleaseInfoViewModel.kt

```diff
@@ -27,14 +27,25 @@ class NewReleaseInfoViewModel @AssistedInject constructor(
         routeToScreen.postValue(NewReleaseInfoNavigationEvents.CloseScreen)
     }
 
-    fun getItems(titles: Array<String>, bodies: Array<String>): List<NewReleaseInfoItem> {
-        if (titles.size != bodies.size) {
-            Timber.e("R.array.new_release_title and R.array.new_release_body must have the same size!")
+    fun getItems(
+        titles: Array<String>,
+        bodies: Array<String>,
+        linkifiedLabels: Array<String>,
+        linkTargets: Array<String>
+    ): List<NewReleaseInfoItem> {
+        if (titles.size != bodies.size || titles.size != linkifiedLabels.size || titles.size != linkTargets.size) {
+            Timber.e(
```

Comment:
Please add a test for this. Checking the size in different locales.

Reply:
done